### PR TITLE
Fixing remaining WebArchive/Staging links

### PR DESCRIPTION
**Pull request by CalebLangbroek**
This is for #7, only the constitution page for now, I'll fix the other 2 soon.

The sub-points are all unordered lists for now because of the CSS styling (don't know why the ordered lists were removed). After #42 is merged, the styling can be cleaned up.

We'll probably want to go over this at some point and make some adjustments.

### CODE REVIEW

**Review by MykalMachon — COMMENTED** [review]
Looks good to me- 

Once the regulations page is done we can merge this in, but no rush!
I appreciate you taking the time to convert this all to markdown @CalebLangbroek 🎉

**Comment by CalebLangbroek** [comment]
Sounds good, no worries!

File: static/documents/minutes.html [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-        <div class="img" id="header-image"><img src="csaheader.png"></div>
-    </div>
-    <div id="banner" class="article">
-    <p>If you would like to contribute to this website, check out its <a href="https://github.com/ufvcsa/ufvcsa.github.io/">GitHub page</a>!</p>
-
-    </div>
-    <div class="article">
-<h1>Minutes</h1>
-    <ul>
-				<li><a href="http://web.archive.org/web/20180812185102/http://ufvcsa.ca/2015/08/23/september-2015-meeting-minutes-2/" rel="bookmark">September 2015 Meeting Minutes</a></li>
```
Comment (CalebLangbroek) [inline static/documents/minutes.html:35]: Removing this because the links here are empty.

**Comment by MykalMachon** [comment]
Looks good to me! nice work.

**Comment by CalebLangbroek** [comment]
Cool, thanks! I'll update the site later today.

**Pull request merged**